temporalcli/commandsmd/commands.md

```diff
@@ -566,657 +1222,1324 @@ Includes options set for [overlap-policy](#options-set-for-overlap-policy).
+in your Workflow code, use `temporal workflow cancel` instead.
+
+Visit https://docs.temporal.io/visibility to read more about Search Attributes
+and Query creation. See `temporal batch --help` for a quick reference.
 
 #### Options
 
-* `--workflow-id`, `-w` (string) - Workflow Id. Either this or query must be set.
-* `--run-id`, `-r` (string) - Run Id. Cannot be set when query is set.
-* `--query`, `-q` (string) - Start a batch to terminate Workflow Executions with given List Filter. Either this or
-  Workflow Id must be set.
-* `--reason` (string) - Reason for termination. Defaults to message with the current user's name.
-* `--yes`, `-y` (bool) - Confirm prompt to perform batch. Only allowed if query is present.
+* `--workflow-id`, `-w` (string) -
+  Workflow ID.
+  You must set either --workflow-id or --query.
+* `--query`, `-q` (string) -
+  Content for an SQL-like `QUERY` List Filter.
+  You must set either --workflow-id or --query.
+* `--run-id`, `-r` (string) -
+  Run ID.
+  Can only be set with --workflow-id.
+  Do not use with --query.
+* `--reason` (string) -
+  Reason for termination.
+  Defaults to message with the current user's name.
+* `--yes`, `-y` (bool) -
+  Don't prompt to confirm termination.
+  Can only be used with --query.
 
-### temporal workflow trace: Terminate Workflow Execution by ID or List Filter.
+### temporal workflow trace: Workflow Execution live progress
 
-The `temporal workflow trace` command display the progress of a [Workflow Execution](/concepts/what-is-a-workflow-execution) and its child workflows with a trace.
-This view provides a great way to understand the flow of a workflow.
+Display the progress of a Workflow Execution and its child workflows with a
+real-time trace. This view helps you understand how Workflows are proceeding:
 
-Use the options listed below to change the behavior of this command.
+```
+temporal workflow trace \
+    --workflow-id YourWorkflowId
+```
 
 #### Options
 
-* `--fold` (string[]) - Statuses for which Child Workflows will be folded in (this will reduce the number of information fetched and displayed). Case-insensitive and ignored if no-fold supplied. Available values: running, completed, failed, canceled, terminated, timedout, continueasnew.
-* `--no-fold` (bool) - Disable folding. All Child Workflows within the set depth will be fetched and displayed.
-* `--depth` (int) - Depth of child workflows to fetch. Use -1 to fetch child workflows at any depth. Default: -1.
-* `--concurrency` (int) - Number of concurrent workflow histories that will be requested at any given time. Default: 10.
+* `--fold` (string[]) -
+  Statuses for folding away Child Workflows.
+  This reduces info fetched and displayed.
+  Case-insensitive.
+  Ignored if --no-fold supplied.
+  Available values: running, completed, failed, canceled, terminated,
+  timedout, continueasnew.
+  Can be passed multiple times.
+  Each fold reduces the amount of information fetched and displayed.
```

Comment:
You say this twice (see a few lines above).

Reply:
Will fix and resolve. Thanks.